dev_tools/docs/build_api_docs.py

```diff
@@ -29,19 +34,21 @@
 from tensorflow_docs.api_generator import public_api
 
 import cirq
+import cirq_google
+
 from cirq import _doc
 
-flags.DEFINE_string("output_dir", "/tmp/cirq_api", "Where to output the docs")
+flags.DEFINE_string("output_dir", "docs/api_docs", "Where to output the docs")
 
 flags.DEFINE_string(
     "code_url_prefix",
-    "https://github.com/quantumlib/cirq/tree/master/cirq",
+    "https://github.com/quantumlib/Cirq/blob/master",
     "The url prefix for links to code.",
 )
 
 flags.DEFINE_bool("search_hints", True, "Include metadata search hints in the generated files")
 
-flags.DEFINE_string("site_path", "quark/cirq/api_docs/python", "Path prefix in the _toc.yaml")
+flags.DEFINE_string("site_path", "reference/python", "Path prefix in the _toc.yaml")
```

Comment:
What prompted this change? At first glance it doesn't seem related to either of the linked issues.

Reply:
This is non-material, because the internal caller overrides this anyway with the correct string, but it was an outdated default value (we had this URL structure when we first had the internal staging site hosted on tensorflow's site) that I just updated. I'll include that in the PR description.